lib/string_reverse.rb

```diff
@@ -1,6 +1,18 @@
 # A method to reverse a string in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n), where n is the number of elements in the array

```

Comment:
Teeny nit: `array`, or `string`?

(Aside: in some lower-level languages like C, there is no `string` type and _character arrays_ are used instead. Some higher level languages also offer [functions that convert strings into character arrays](https://www.javatpoint.com/java-string-tochararray).)